next-env.d.ts

```diff
@@ -0,0 +1,6 @@
+/// <reference types="next" />
```

Comment:
Should this file get committed?

Reply:
it should, the docs say you shouldn't remove it and some of the settings in this file prevent eslint errors

Reply:
👍